SCP fightclub 3d/Assets/Scripts/Movement.cs

```diff
@@ -10,14 +10,14 @@ public class Movement : MonoBehaviour
     [SerializeField]
     private float speed = 1.0f;
 
-    private new Rigidbody2D rigidbody;
+    private new Rigidbody rigidbody;
 
     public bool isPlayer2 = false;
     public bool isPlayer1 = false;
 
     void Start()
     {
-        rigidbody = GetComponent<Rigidbody2D>();
+        rigidbody = GetComponent<Rigidbody>();
```

Comment:
I don't have much experience with 2D and I'm not sure how converting things over to 3D would work, but this looks right for changing a rigidbody.